samples/matter/light_bulb/CMakeLists.txt

```diff
@@ -6,6 +6,16 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
+option(BUILD_WITH_DFU "Build target with Device Firmware Upgrade support" OFF)
+if(BUILD_WITH_DFU)
+    set(OVERLAY_CONFIG ${CMAKE_CURRENT_SOURCE_DIR}/../common/config/overlay-dfu_support.conf)
+    set(mcuboot_CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/child_image/mcuboot.conf)
```

Comment:
I understand why we need to to that for Thingy (i.e. Thingy53-specific configuration in mcuboot does not exist), but I wonder if it's valuable to copy-paste all the mcuboot configuration for remaining samples instead of relying on defaults. We can just use overlays if we need to adjust some settings. Any thoughts?

Reply:
I don't have strong opinion on that. I thought about potential inconsistencies between our samples that may appear, as in weather-station we will use some configs controlled by us and in other samples we will use some configs that may change without our control and knowledge. Nevertheless if we are going to move weather station to applications, I think it will start gradually differentiate from other samples anyway, so I also don't have objections to follow your suggestions.